tests/run_tests.sh

```diff
@@ -9,5 +9,10 @@
 # agreement to the Shotgun Pipeline Toolkit Source Code License. All rights 
 # not expressly granted therein are reserved by Shotgun Software Inc.
 
+#
+# Allows to run the unit tests against the python executable. The actual version is determined
+# by the user's environment.
+#
+
 find . -name "*.pyc" -delete
-python -3 `dirname $0`/run_tests.py $*
+python `dirname $0`/run_tests.py $*
```

Comment:
shoudn't we add a `-3` argument to the travis script now if re move it here? So we still get those warnings reported by CI?

Reply:
i just tried it and i guess it won't matter - the travis py3 task takes care of it completely separately!